lang/en.json

```diff
@@ -71,9 +71,14 @@
     "WITCHER.StSpd": "Speed",
     "WITCHER.StVigor": "Vigor",
 
+    "WITCHER.Actor.Name": "Character Name",
     "WITCHER.Actor.Race": "Race",
     "WITCHER.Actor.Gender": "Gender",
     "WITCHER.Actor.Age": "Age",
+    "WITCHER.Actor.Perks": "Perks",
+
+    "WITCHER.Actor.Adda.Profession": "Add a Profession",
```

Comment:
What does Adda Stands for?

Reply:
Add a :P could probably change it to add. Looks a bit nicer maby